middleware/tls/README.md

```diff
@@ -0,0 +1,13 @@
+# tls
```

Comment:
ok, i see the tls stuff is here so no need for it like I talked about above. I still think though if you leave the server definition as just domain:hostport we are better off. Otherwise you could do dns-tls and grpc for example and they would look like different server blocks but in fact are on the same port and so need to share the TLS info.